answers/Interfaces/b447ca5d-c0f2-4664-8e83-ca91f39a71be/Mammal.php

```diff
@@ -0,0 +1,25 @@
+<?php
+require_once(__DIR__ . '/Animal.php');
+require_once(__DIR__ . '/LifeCycle.php');
+
+class Mammal extends Animal implements LifeCycle
+{
+    public function species()
+    {
+        return "animal";
+    }
+
+    public function stage()
+    {
+        return "adult";
+    }
+
+    public function display()
+    {
+        echo "The " . parent::getType() . " is an " . $this->stage() . " " . $this->species() . ".";  
```

Comment:
Same as the other lesson, if we're not overwriting the `getType` method, there's no need to call `parent` since it will traverse up to it's hierarchy tree to find the method.